### PR TITLE
New(robot): Environment variable TEST_PORT for setting test port

### PR DESCRIPTION
**Pull request by rkubis**
## Description

There is environment variable called `TEST_MR_HOST` for Robot Framework tests setting Model Registry host that can be used for testing Model Registry instance deployed anywhere, but there is not any environment variable for setting port of the instance for testing it.

This PR provides new environment variable called `TEST_MR_PORT` for setting test port of Robot Framework tests.

## How Has This Been Tested?

1. Deploy Model Registry instance anywhere with publicly reachable host and port.
2. Run Robot Framework tests against it with setting TEST_MR_HOST and TEST_MR_PORT environment variable:
```TEST_MR_HOST=<model-registry-hostname> TEST_MR_PORT=<model-registry-port> robot test/robot/UserStory.robot```

## Merge criteria:

- [x] The commits and have meaningful messages; the author will squash them [after approval](https://github.com/opendatahub-io/opendatahub-community/blob/main/contributor-cheatsheet.md#:~:text=Usually%20this%20is%20done%20in%20last%20phase%20of%20a%20PR%20revision) or will ask to merge with squash.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: test/robot/MRkeywords.resource [hunk 1]
```diff
@@ -6,14 +6,15 @@ Library    ModelRegistry.py
 
 *** Variables ***
 ${MR_HOST}     %{TEST_MR_HOST=localhost}
+${PORT}        %{TEST_MR_PORT=8080}
```
Comment (tarilabs) [inline test/robot/MRkeywords.resource:9]: just for consistency, what do you think of prefixing with `MR_` like for the other variable?

```suggestion
${MR_PORT}        %{TEST_MR_PORT=8080}
```

and related needed modifications in the below of the script?

**Review by tarilabs — COMMENTED** [review]
thank you @rkubis for your PR ! 👏  this is very helpful to make it configurable

File: test/robot/MRkeywords.resource (hunk 1 above)
Comment (rkubis) [reply test/robot/MRkeywords.resource]: @tarilabs Thanks for suggestion, good catch 👍  Updated.

**Review by rareddy — APPROVED** [review]
/approve

**Pull request merged**